### ISSUE TITLE
`WriteSection` does not use the outline

### ISSUE THREAD

**Issue by s7ev3n** [stanford-oval/storm#43]
Hi, thanks for the great work !

I notice that when generating section, we give the children subsections within of a first level section ([`section_outline = "\n".join(queries_with_hashtags)`](https://github.com/stanford-oval/storm/blob/463634e640e516734a1de30f52701891855f8dcd/src/storm_wiki/modules/article_generation.py#L90))and also pass it to `gen_section`, however, this information is not used, as in the `WriteSection` prompt does not use this outline. That may indicate that the section generation does not follow the generated outline strictly (the first level section is there but the subsections are all the llm improvisation).

**Comment by Yucheng-Jiang** [comment]
@s7ev3n thanks for your interest in our work.

Yijia provides a detailed explanation of our design rationale in #30. Does it answer your question?

TL;DR 
>We use very detailed hierarchical outline as a way to organize the collected information, so we can find the information out of a large pool to write a specific section rather than feeding all the collected information to the LM.

The real outline for each section presented to readers can be **customized as needed**.  In our paper implementation, the presented deliverable is more like Wikipedia-like article, which prefers longer, more coherent paragraphs.

**Comment by s7ev3n** [comment]
@Yucheng-Jiang Thanks! That exactly answers my question.
